pkg/controller/compliancesuite/compliancesuite_controller.go

```diff
@@ -147,6 +147,13 @@ func (r *ReconcileComplianceSuite) Reconcile(ctx context.Context, request reconc
 		return reconcile.Result{}, r.issueValidationError(suite, errorMsg, reqLogger)
 	}
 
+	if r.isComplianceSuiteSuspended(suite) {
```

Comment:
Revert this since we're going to do this in the scansettingbinding controller instead.